cvat-ui/src/components/annotation-page/standard3D-workspace/controls-side-bar/photo-context.tsx

```diff
@@ -0,0 +1,39 @@
+// Copyright (C) 2021 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import React from 'react';
+import Icon from '@ant-design/icons';
+import Tooltip from 'antd/lib/tooltip';
+
+import { CameraIcon } from 'icons';
```

Comment:
import { CameraOutlined } from '@ant-design/icons';
We do not need this icon as an asset because we can use it directly from antd.
Please, remove it from the project.

Reply:
Resolved